tests/python/relay/test_pass_gradient.py

```diff
@@ -27,6 +29,20 @@
 import tvm.relay.op as op
 
 
+def count_ops(expr):
```

Comment:
can you move this to some common file? this look like something useful to other places as well.

Reply:
I'll move it to `python/tvm/relay/testing/__init__.py` when rebasing post #5946 .